### ISSUE TITLE
Remove forked act_runner

### ISSUE THREAD

**Issue by dboreham** [cerc-io/stack-orchestrator#250]
Upstream fixes in our forked act_runner then revert to building our container from the upstream repo.

**Comment by zramsay** [comment]
running a combo of instructions from the #238 description and https://github.com/cerc-io/stack-orchestrator/blob/main/app/data/stacks/build-support/README.md
worked, with the caveat that #240 was only a partial solution; I had to either modify the `stack.yml` or `mv ~/cerc/act_runner ~/cerc/act-runner`

**Comment by dboreham** [comment]
> running a combo of instructions from the #238 description and https://github.com/cerc-io/stack-orchestrator/blob/main/app/data/stacks/build-support/README.md worked, with the caveat that #240 was only a partial solution; I had to either modify the `stack.yml` or `mv ~/cerc/act_runner ~/cerc/act-runner`

I think this comment is a different bug. Are you saying that the act-runner container didn't build?

**Comment by zramsay** [comment]
You're right, my comment is ~ tangential to this issue. No, it built fine after I made either one of those tweaks; just needed a logical place to track my hack

**Comment by dboreham** [comment]
But your hack must mean there's a bug, right?

**Comment by dboreham** [comment]
It shouldn't be necessary to rename the cloned repo, so something is wrong somewhere..

**Comment by dboreham** [comment]
Repo has the correct name here: https://github.com/cerc-io/stack-orchestrator/blob/main/app/data/stacks/package-registry/stack.yml#L6
And then it gets built here: https://github.com/cerc-io/stack-orchestrator/blob/main/app/data/container-build/cerc-act-runner/build.sh#L3 from that cloned repo to make a container tagged "act-runner".

**Comment by zramsay** [comment]
Right, so this is a case of me being uncertain if it's a) user error, b) incomplete existing documentation, c) undocumented but known bug w/ fix incoming, or d) unknown bug that is now documented.

**Comment by dboreham** [comment]
Perhaps you had a torn set of commits in your S-O? I think I made two merge commits to fix all the _ vs - stuff.

**Comment by zramsay** [comment]
could very well be, or too much "trying to get something to work" by mucking about in `~/.shiv/laconic-so_BLAH/...`

**Comment by dboreham** [comment]
Closing since this hasn't reoccurred.
